### DISCUSSION TITLE
Corrupted video while recording videos using "PlusServerRemoteControl.exe"

### DISCUSSION THREAD

**Discussion by lbo2021** [PlusToolkit/PlusLib#900] [Q&A]
Hi,

I noticed strange behaviors if I record a video for some minutes using "PlusServerRemoteControl.exe". After some minutes the recording is corrupted. The issue I encountered are:
- Wrong colors after some minutes
- Image transformed

Please see at this link a video that show the issues I encountered https://drive.google.com/file/d/1lNBy-w3FqKyTLfSMq6tnbMu8NuuLVOU1/view?usp=sharing (from minute 1.10)

This video is a replay on Slicer of a video recorded using "PlusServerRemoteControl.exe". Please look at the last seconds.
The issue is generated during the recording process. I'm sure that the input video is good because while I was recording I was at the same time connected to the Plus Server with Slicer, so I could check the video and it did not have any error.

I tried to record in different formats like .mkv or .nrrd but the issue is always present.

Do you know how to solve this issue?

**Comment by adamrankin** [comment]
It looks like the red and blue colour channels have been switched. I don't know why this might happen. @lassoan @Sunderlandkyl any ideas? OpenIGTLink issue?

**Reply by lbo2021** [reply]
the live video is correctly displayed in Slicer via OpenIGTLink. The problem lies in the recording using "PlusServerRemoteControl.exe". 

The video I have attached is a recording made with OBS of a replay on Slicer of a video recorded using "PlusServerRemoteControl.exe".

In the live video these kind of colors changes or image traslations are not present. They are only present in the recorded video

**Reply by adamrankin** [reply]
Right. Are there any messages/warnings/errors in the log associated with the recording?

**Reply by lbo2021** [reply]
No message or warning or error
